src/main/webapp/script.js

```diff
@@ -193,15 +197,6 @@ async function fetchQuestions(page) {
   });
 }
 
-/**
- * Fetches a single question and its answers from server, 
- * wraps each in an <li> element, and adds them to the DOM.
- */
-async function fetchQuestionAndAnswers() {
-  const response = await fetch('/answers');
-  const questionsObject = await response.json();
-}
-
```

Comment:
Is this not longer going to be used?

Reply:
nope, I found it easier to have two separate fetch functions to create the elements in the corresponding order